pkgs/tools/games/minecraft/hmcl/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib, stdenv
+, fetchurl
+, jre
+, makeWrapper
+, makeDesktopItem
+, copyDesktopItems
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hmcl";
+  version = "3.5.3.223";
+
+  src = fetchurl {
+    url = "https://github.com/huanghongxun/HMCL/releases/download/v${version}/HMCL-${version}.jar";
+    sha256 = "sha256-8g2FMvAiAKfxJUY0G7wl6d44wOpVsknFHGZ85IkOzFc=";
+  };
+
+  icon = fetchurl {
+    url = "https://aur.archlinux.org/cgit/aur.git/plain/craft_table.png?h=hmcl-bin";
```

Comment:
Could this be fixed to a specific commit instead? If they update the image on the AUR our hash gets invalidated now.

Reply:
Looks like an icon is provided inside of the repository, you should use that instead. See this [ico file](https://github.com/huanghongxun/HMCL/blob/3da0c1d8e3339f2713857fa2ee962b8de6212fc1/HMCLauncher/HMCL/HMCL.ico), it can be converted to a PNG using `icotool`.

Reply:
I can't find `icotool` this tool

Reply:
> Looks like its included with the `icoutils` package in nixpkgs

Before you replied, I found imagemagick, it just is command `convert`